### ISSUE TITLE
Android - Accessibility - Pick Language Screen - TalkBack is Reading Content Many times

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#201]
While reading with TalkBack at the top of the screen in continuous reading mode, it reads the ILAB text, DOL seal ALT text, and the text below it.

Then it reads all that again text again followed by the English button. 

Next, TalkBack reads the beginning text and the English and Spanish buttons.

Then TalkBack keeps reading that pattern up over for French and Malay.

Not sure why TalkBak is not reading down the page once in continuous reading mode.

**Comment by pbhatt17** [comment]
@jfc3-dol 
Please retest this issue.

**Comment by jfc3-dol** [comment]
@pbhatt17 

To do that test, I will first have to delete the application and reload it on my devices. So I would like to finish at least first the other Android issues ILAB sent me to test. 

On the Bookmark page, when reading in continuous reading mode (swipe "L" then choose read from current location), TalkBack is reading the page repeatedly. So I'm guessing it might be an issue there and in other places. I need to test to find out for sure.

**Comment by jfc3-dol** [comment]
@pbhatt17 

After finishing the testing of the other Android issues, I uninstalled the Comply Chain applications. I then retested the language page with TalkBack, and it only read through the content once.

So I'm going to close this issue.

NOTE - TalkBack did read the three non-English language buttons in English instead of the language of the button. I'm going to look to re-open an issue we previously closed related to this problem.
